src/main/java/com/uid2/shared/attest/AttestationTokenService.java

```diff
@@ -1,22 +1,42 @@
 package com.uid2.shared.attest;
 
 import java.time.Instant;
+import java.time.temporal.ChronoUnit;
 
 public class AttestationTokenService implements IAttestationTokenService {
 
     private final String encryptionKey;
     private final String encryptionSalt;
+    private final long expiresAfterSeconds;
 
+    @Deprecated
     public AttestationTokenService(String encryptionKey, String encryptionSalt) {
         this.encryptionKey = encryptionKey;
         this.encryptionSalt = encryptionSalt;
+        this.expiresAfterSeconds = 2 * 3600; // 2 hours;
     }
 
+    public AttestationTokenService(String encryptionKey, String encryptionSalt, long expiresAfterSeconds) {
+        this.encryptionKey = encryptionKey;
+        this.encryptionSalt = encryptionSalt;
+        this.expiresAfterSeconds = expiresAfterSeconds;
+    }
+
+    @Override
+    public String createToken(String userToken) {
+        Instant expiresAt = Instant.now().plus(this.expiresAfterSeconds, ChronoUnit.SECONDS);
```

Comment:
We probably should do what `uid2-admin` was doing and create a wrapper around `Instant.now()`, but that's for future usage :)

Reply:
Thanks, let me keep this change small and upgrade everything in one go when we need to.